shuffle/cli/tests/shuffle-forge-cli-tests.rs

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+mod common;
+
+use forge::{
+    forge_main, AdminContext, AdminTest, ForgeConfig, LocalFactory, Options, Result, Test,
+};
+
+use common::{bootstrap_shuffle, ShuffleTestHelper};
+use smoke_test::scripts_and_modules::enable_open_publishing;
+use std::{os::unix::prelude::ExitStatusExt, process::ExitStatus};
+use tempfile::tempdir;
+
+const HOME_PATH: &str = "--home-path";
```

Comment:
This const adds a level of indirection that I don't think is helpful to a new dev. And its usage is inconsistent with other flags.

```rust
        let output = std::process::Command::new(BINARY)
            .args([
                HOME_PATH, <-- flag
                base_path_string.as_str(),
                "transactions",
                "--network", <--- flag
                "forge",
            ])
```

I would get rid of it and just pass `--home-path` inline like --network.

Reply:
got it!